app/views/govuk_publishing_components/components/docs/attachment.yml

```diff
@@ -0,0 +1,53 @@
+name: Attachment (experimental)
+description: Displays a link to download an attachment and metadata about the file
+body: |
+  This component is marked as experimental as it is part of a drive to provide
+  a consistent place for attachment presentation logic (currently this logic is
+  within [Govspeak][] and [Whitehall][]). The API for this
+  component may change during this process.
+
+  It is used to show data about an attachment document to GOV.UK content.
+
+  [Govspeak]: https://github.com/alphagov/govspeak/blob/c3a0742c87537a371108d097cea23688efa90a14/lib/govspeak/presenters/attachment_presenter.rb
+  [Whitehall]: https://github.com/alphagov/whitehall/blob/5c760eea912320c5a80ef59791df47161d889f2f/app/helpers/document_helper.rb
+accessibility_criteria: |
+  The image must:
+
+  * have alt text
```

Comment:
Whitehall has a precedent of these not having alt text and instead using ARIA hidden to not show that to users. So I think here we'd say that we don't allow targetting an image and mark them as hidden from screen readers.

Reply:
Now removed